libs/ui-lib/lib/ocm/services/ClusterDetailsService.ts

```diff
@@ -70,6 +69,12 @@ const ClusterDetailsService = {
       params.pullSecret = values.pullSecret;
     }
 
+    if (values.platform) {
```

Comment:
This will need some adjustments since `none` is not always the correct option. Sometimes it needs to be `baremetal`, so when going from a integration to "no integration", the PATCH is either `none` or `baremetal`.

Reply:
@celdrake and how do we know that we need baremetal?

Reply:
We'll need to check with the backend. 
Since it's possible to go from having an enabled integration to not having one, we need to know which is the correct value in each case. Otherwise it won't be possible to remove the integration.

Reply:
I'll check it how it was being done up until now.

Reply:
@celdrake this is the code that we are using now:
https://github.com/openshift-assisted/assisted-installer-ui/blob/master/libs/ui-lib/lib/ocm/services/HostDiscoveryService.ts#L10-L22.

But now it seems that we have to remove UMN according to this ticket: https://issues.redhat.com/browse/MGMT-14936

We need to BE to implement these changes

Reply:
I think it's best to handle those changes separately, otherwise this story will be blocked until those other changes are implemented on the BE, and then it would be a even larger MR.

You just need to include the code that decides if it needs to PATCH with `none` or `bareMetal` according to the current rules which you pointed out, in `HostDiscoveryService`